public/css/showoff.css

```diff
@@ -783,31 +774,23 @@ body#download ul#downloads li {
 
 /** Print **/
 @media print {
-  #sidebarWrapper,
```

Comment:
hamburger doesn't appear in any generated pdf.  is it being removed by courseware manager?